### ISSUE TITLE
Невозможность пролистывать подряд несколько страниц

### ISSUE THREAD

**Issue by GoogleCodeExporter** [JustLoy77/ebookdroid#686]
```
В версиях 2 и выше изменился способ смены 
страниц с помощью клавишь громкости. В 
частности, если нажимать подряд несколько 
раз кнопку "громкость -", то в какой-то 
момент программа перестает на нее 
реагировать (перелистывать страницы) до 
тех пор, пока не отобразится текущая 
страница (кроме того, бывают случаи, что 
программа вообще зависает на этом процессе 
и приходится закрывать и снова открывать 
книгу). В версиях до 2 я мог сколь угодно 
много раз нажимать эту кнопку и счетчик 
страниц при этом увеличивался без проблем 
(в итоге отображалась нужная страница). 
Сейчас же если я знаю, что мне надо 
прокрутить 10 страниц вперед, я не могу 
просто тупо нажать 10 раз кнопку. Из-за этого 
косяка долгое время пользовался платной 
версией (купил ее, чтобы поддержать 
разработчиков программы, которая так 
нравилась, а через некоторое время вышла 
версия 2 ebookdroid и все стало печально), не 
обновляя ее. Потом случайно обновил, да и к 
тому же купил другой планшет (на нем уже 
вынужденно скачалась новая версия). В итоге 
там оказалась такая же тормознутая схема 
смены страниц. Как быть? (Долго не писал Вам 
об этом в надежде, что сами заметите и 
исправите, но, видимо, не срослось)
```

Original issue reported on code.google.com by `chugunov...@gmail.com` on 3 Jan 2014 at 9:23

**Comment by GoogleCodeExporter** [comment]
```
Хм. Действительно не обращали внимание. Это 
побочный эффект, когда мы боролись со 
спонтанными перелистываниями. Попробуйте 
отключить анимацию листания в настройках, 
Все настройки/Нажатия и 
прокрутка/Анимировать прокрутку. А также, 
если пользуетесь одностраничным режимом 
поставьте пустой эффект листания. Должно 
помочь.

К сожалению параллельная с анимацией 
обработка листания приводила к случайным 
перебросам в начало книги.

Спасибо, что наконец написали. На будущее - 
если что-то заметили, пишите сразу.
```

Original comment by `Andrei.K...@gmail.com` on 3 Jan 2014 at 9:35
- Changed state: **Discussion**

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
Пользуюсь одностраничным режимом. 
Анимация перелистывания (все настройки - 
отрисовка - анимация) была отключена Сейчас 
также отключил Все настройки/Нажатия и 
прокрутка/Анимировать прокрутку. Не 
помогло (даже не увеличивает счетчик 
страницы, пока текущая не отобразится). 
Понажимал и он даже опять завис на книге
```

Original comment by `chugunov...@gmail.com` on 3 Jan 2014 at 10:03

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 2.1.1
```

Original comment by `Alexander.V.Kasatkin@gmail.com` on 9 Jan 2014 at 8:19
- Changed state: **Fixed**
